### DISCUSSION TITLE
Setting DataGridRow.IsVisible does not work after DataGrid is sorted by column

### DISCUSSION THREAD

**Discussion by arnirichard** [AvaloniaUI/Avalonia#13655] [Q&A]
I am setting `DataGridRow` visibility as follows

`<DataGrid.Styles>

    <Style Selector="DataGridRow">
    
	    <Setter Property="IsVisible">
	    
		    <Setter.Value>
		    
			    <MultiBinding Converter="{StaticResource DataGridRowVisibilityConverter}">
			    
				    <Binding/>
				    
				    <Binding Path="SelectedItem" ElementName="someCombo"/>
				    
			    </MultiBinding>
			    
		    </Setter.Value>
		    
	    </Setter>
	    
    </Style>
    
</DataGrid.Styles>`

This works until I have sorted column by pressing the column header then it stops working. Why?

**Comment by timunie** [comment]
I think it's better to filter the ItemsSource tbh. For Example DynamicData can do, or DataGridCollectionView can do,
